backend/queues/makeOfferProcessor.js

```diff
@@ -62,7 +64,7 @@ async function processor(job) {
     job.log(str)
     job.progress(progress)
   }
-  const jobId = `${job.queue.name}-${job.id}` // Prefix with queue name since job ids are not unique across queues.
+  const jobId = `${get(job, 'queue.name', '')}-${job.id}` // Prefix with queue name since job ids are not unique across queues.
```

Comment:
I think this was failing for you because you were using the in-memory queue vs redis.
Sorry I think I added this log.
It might be more elegant to fix it by adding a queue field to the job object here:
https://github.com/OriginProtocol/dshop/blob/master/backend/queues/fallbackQueue.js#L35
Like
```
    const job = {
      queue: this
      data: data,
      progress: () => undefined,
      log: log.info
```